libs/application/templates/parental-leave/src/lib/messages.ts

```diff
@@ -1531,12 +1597,72 @@ export const errorMessages = defineMessages({
     defaultMessage: 'Vinsamlegast veldu lokadagsetningu',
     description: 'End date can not be empty',
   },
+  periodsEndDateTooLate: {
+    id: 'pl.application:answerValidators.periodsEndDateTooLate',
+    defaultMessage:
+      'Endadagsetning getur ekki verið meira en 24 mánuðum eftir áætlaðan fæðingardag',
+    description:
+      'Copy for when end date is more than 24 months away from expected date of birth',
+  },
   periodsRatio: {
     id: 'pl.application:answerValidators.periodsRatio',
     defaultMessage:
       'Lágmarkið er {minPeriodDays} dagar í orlofi, þú hefur valið {diff} daga á {ratio}% sem endar sem aðeins {diffWithRatio} daga leyfi.',
     description: 'Copy when ratio is invalid',
   },
+  periodsRatioDaysMissing: {
+    id: 'pl.application:answerValidators.periodsRatioDaysMissing',
+    defaultMessage: 'Ekki tókst að reikna út fjölda daga fyrir tímabil',
+    description:
+      'Copy when days to be used by period is missing (caused by API not being able to calculate)',
```

Comment:
"caused by API not being able to calculate" can all be removed here and further down